source/css/_common/components/sidebar/sidebar-toggle.styl

```diff
@@ -25,6 +25,6 @@
   width: 100%;
   background: white;
   margin-top: 3px;
-
+  transition: all 0.4s;
```

Comment:
Is this userful when `motion.enable`?

```css
if not hexo-config('motion.enable') {
  transition: all 0.4s;
}
```

Reply:
I have changed the logic, `transition: all 0.4s;` is always needed now